npm/ng-packs/packages/components/extensible/src/lib/utils/typeahead.util.ts

```diff
@@ -39,11 +39,11 @@ export function createTypeaheadOptions(
 }
 
 export function getTypeaheadType(lookup: ExtensionPropertyUiLookupDto, name: string) {
-  return Boolean(lookup.url)
-    ? ePropType.Typeahead
-    : name.endsWith(TYPEAHEAD_TEXT_SUFFIX)
-    ? ePropType.Hidden
-    : undefined;
+  if (!!lookup.url) {
+    return ePropType.Typeahead;
+  } else {
```

Comment:
We don't need else block

```ts
if (!!lookup.url) {
    return ePropType.Typeahead;
}

return name.endsWith(TYPEAHEAD_TEXT_SUFFIX) ? ePropType.Hidden : undefined;
```

Reply:
I don't add any logic. The logic already exists. I just convert ternary to readable version of if.